File: app/src/main/java/ir/touristland/Models/HotelDetail.java
package ir.touristland.Models;

import java.util.HashMap;
import java.util.Map;

/**
 * Created by ZAMAN on 5/22/2018.
 */

public class HotelDetail {
    private static final HotelDetail ourInstance = new HotelDetail();
    public int StartDate = 1;
    public int EndDate = 0;
    public int Destination = 0;

    public Map<String, String> params = new HashMap<>();

    private HotelDetail() {
    }

    public static HotelDetail getInstance() {
        return ourInstance;
    }


    public Map<String, String> getParams() {
        return params;
    }

    public void setParams(Map<String, String> params) {
        this.params = params;
    }
}

File: app/src/main/java/ir/touristland/Models/HotelDetail.kt
package ir.touristland.Models

import java.util.HashMap

/**
 * Created by ZAMAN on 5/22/2018.
 */

class HotelDetail private constructor() {
    var StartDate = 1
    var EndDate = 0
    var Destination = 0

    var params: Map<String, String> = HashMap()

    companion object {
        val instance = HotelDetail()
    }
}
